FurEver/API Data/DogBreed.cs

```diff
@@ -17,14 +17,14 @@ public class DogBreedService
 
         public static async Task<List<DogBreed>> FetchBreedsAsync()
         {
-            var constants = Constant.Instace;
-            var dogBreedsAPI = constants.DogBReedsAPI;
+            var config = Constant.Instace;
+            var dogBreedsAPI = config.DogBReedsAPI;
 
             var apiClient = new HttpClient();
             try
             {
                 HttpResponseMessage response = await apiClient.GetAsync(dogBreedsAPI);
-                Console.WriteLine(response);
+                Console.WriteLine($"API Response Status: {response.StatusCode}");
```

Comment:
It's still Console.WriteLine, though.  Go one step further and use true logging with severity levels.
https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line